interfaces/cython/cantera/thermo.pyx

```diff
@@ -660,6 +660,11 @@ cdef class ThermoPhase(_SolutionBase):
         def __get__(self):
             return self._getArray1(thermo_getMolecularWeights)
 
+    property species_charges:
+        """Array of species charges [-]."""
```

Comment:
The description of units should be updated to reflect that they are in units of the elementary charge.